### ISSUE TITLE
Mismatch in accuracy

### ISSUE THREAD

**Issue by tejank10** [microsoft/robust-models-transfer#3]
Hello!

Great work, thanks for making the code public!

I tried running [L2 PGD trained Resnet50 eps1](https://github.com/microsoft/robust-models-transfer#download-our-robust-imagenet-models) model ImageNet validation set. I was getting 82.5% as standard top-1 accuracy vs 70.43% as mentioned in the paper. The robust accuracy was 69.4%.
Robustness library was used for evaluation with the following command from CLI. Attack specs are L2 PGD-3 with eps=1 step size=2/3 as mentioned in the paper.
```
python -m robustness.main --dataset imagenet --data ../../../datasets/imagenet2012/ --adv-train 1 --arch resnet50 --eps 1. --attack-lr 0.666666667 --attack-steps 3 --constraint 2 --eval-only 1 --out-dir logs --resume ../../pretrained_models/resnet50_l2_eps1.ckpt --adv-eval 1 --batch-size 128 --mixed-precision 1
```

I am not sure what am I missing here. Can you please suggest what factors should I take care of while running an evaluation for it? Thank you.

**Comment by andrewilyas** [comment]
Hi @tejank10 ! Can you make sure that your ImageNet dataset is properly formatted (PyTorch ImageFolder format)? I just ran the command you sent and got the following:
```
/tmp|⇒  python -m robustness.main --dataset imagenet --data /scratch/engstrom_scratch/imagenet --adv-train 0 --arch resnet50 --eps 1. --attack-lr 0.666666667 --attack-steps 3 --constraint 2 --eval-only 1 --out-dir logs --resume resnet.pt --adv-eval 1 --batch-size 128
Logging in: /tmp/logs/3ed6f1d8-63b5-4ea4-8d74-53d763feaa4e
==> Preparing dataset imagenet..
=> loading checkpoint 'resnet.pt'
=> loaded checkpoint 'resnet.pt' (epoch 90)
{
  "config_path": null,
  "eval_only": 1,
  "exp_name": null,
  "dataset": "imagenet",
  "data": "/scratch/engstrom_scratch/imagenet",
  "arch": "resnet50",
  "batch_size": 128,
  "workers": 30,
  "resume": "resnet.pt",
  "resume_optimizer": 0,
  "data_aug": 1,
  "mixed_precision": 0,
  "out_dir": "logs",
  "epochs": null,
  "lr": null,
  "weight_decay": null,
  "momentum": null,
  "step_lr": null,
  "step_lr_gamma": null,
  "custom_lr_multiplier": null,
  "lr_interpolation": null,
  "adv_train": 0,
  "adv_eval": 1,
  "log_iters": null,
  "save_ckpt_iters": null,
  "attack_steps": 3,
  "constraint": "2",
  "eps": "1.",
  "attack_lr": "0.666666667",
  "use_best": 1,
  "random_restarts": 0,
  "random_start": 0,
  "custom_eps_multiplier": null,
  "version": "1.2.1.post1"
}
Val Epoch:0 | Loss 1.2049 | NatPrec1 70.428 | NatPrec5 89.414 | Reg term: 0.0 ||: 100%|██████████████| 391/391 [00:59<00:00,  6.61it/s]
```

I checked this against PyTorch 1.6 and 1.7.

**Comment by tejank10** [comment]
Thanks, that helped!
